website/pages/blog/announcing-free-single-sign-on-for-graphql-hive.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing free Single Sign On for GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: laurin
+date: 2022-11-04
+description: Link your GraphQL Hive Organization to your OAuth OIDC provider of choice.
+image: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-free-single-sign-on-for-graphql-hive/thumbnail.png
+---
+
+[Last month](/blog/announcing-self-hosted-graphql-hive) we announced self-hosted GraphQL Hive.
+
+Today we are excited to announce that due to popular demand we now support connecting a GraphQL Hive Organization to an OAuth Open ID Connect Provider (OIDC) for Single Sign On (SSO).
+
+> GraphQL Hive is our schema registry solution for continuously evolving your GraphQL schema backed by performance insights and usage statistics. Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
+
+GraphQL Hive gives you the flexibility of choosing your existing identity provider of choice for authenticating users, such as Google Workspaces, Okta, Auth0, and many more.
+
+Users logging into GraphQL Hive via an OIDC provider will be automatically added and limited to interacting within your organization.
+
+You can start using OIDC SSO on the GraphQL Hive Cloud version for any type of organization at no additional cost starting today.
+
+Offering this feature for free is a big deal for us.
+We at The Guild believe that Single Sign On is a core security requirement and not a luxury feature that you have to buy in.
```

Comment:
Should we link to the SSO wall of shame?

Reply:
I think we can!